docs/UserGuide.md

```diff
@@ -99,16 +136,16 @@ To see a list of all your guests, you can use the **`list`** command.
         <br><br>
 
 10. In the event that a guest orders room service, you can add charges to the guest's bill using the **`bill`** command.<br>
-For example, you can charge the third guest `30.35` by running the command below.
-    * **`bill`** `3` `b/+30.35`: Adds `30.35` to the bill of the third guest.
+For example, you can charge the third guest _30.35_ by running the command below.
+    * **`bill`** `3` `b/+30.35`: Adds _30.35_ to the `bill` of the third guest.
       <br><br>
 
 11. When a guest checks out of your hotel, you can remove the guest using the **`delete`** command.
     * **`delete`** `3` : Deletes the third guest shown in the current list.
```

Comment:
Should we add the For example, ... for consistency?

Reply:
Agreed!